app/src/androidTest/java/com/github/ybecker/epforuml/ForumAdapterTest.kt

```diff
@@ -29,29 +29,27 @@ class ForumAdapterTest {
 
     @Test
     fun displayRecyclerTest() {
```

Comment:
I feel like you changed the purpose of this tests? You don't check if the recycler view is in the view any more.